### ISSUE TITLE
am2302 not available for some measurements 

### ISSUE THREAD

**Issue by jonashoechst** [Nature40/pysensorproxy#31]
```

2019-04-04 17:50:01,079 - sensorproxy.app - ERROR - Sensor 'am2302' is not available: No AM2302 instance on pin 4
2019-04-04 18:45:01,406 - sensorproxy.app - ERROR - Sensor 'am2302' is not available: No AM2302 instance on pin 4
2019-04-04 22:00:01,088 - sensorproxy.app - ERROR - Sensor 'am2302' is not available: No AM2302 instance on pin 4
2019-04-04 22:25:01,437 - sensorproxy.app - ERROR - Sensor 'am2302' is not available: No AM2302 instance on pin 4
2019-04-04 22:30:01,384 - sensorproxy.app - ERROR - Sensor 'am2302' is not available: No AM2302 instance on pin 4
2019-04-05 06:25:01,424 - sensorproxy.app - ERROR - Sensor 'am2302' is not available: No AM2302 instance on pin 4
2019-04-05 09:25:00,580 - sensorproxy.app - ERROR - Sensor 'am2302' is not available: No AM2302 instance on pin 4
2019-04-05 09:35:01,206 - sensorproxy.app - ERROR - Sensor 'am2302' is not available: No AM2302 instance on pin 4
2019-04-05 09:40:00,848 - sensorproxy.app - ERROR - Sensor 'am2302' is not available: No AM2302 instance on pin 4
```

**Comment by jonashoechst** [comment]
possible fix: do multiple measurements and use the average / or first measurement

**Comment by jonashoechst** [comment]
multiple measurements are implemented in c5e1977ccc1b2685370b997f1cfb02e9459d6669, but still if one measurement fails, the sensor is not read further
